app/views/hosts/_form.html.erb

```diff
@@ -27,8 +27,8 @@
     <div class="tab-content">
 
       <div class="tab-pane active" id="primary">
-        <%= text_f f, :name, :size => "col-md-5", :value => name_field(@host),
-            :help_inline => _("This value is used also as the host's primary interface name.") %>
+        <%= text_f f, :name, :size => "col-md-4", :value => name_field(@host),
+            :help_inline => link_to_function(icon_text("random", _("Randomize")), "randomizeName()") + ' ' + _("This value is used also as the host's primary interface name.") %>
```

Comment:
I think the link should be hidden when the name generator is off as it does not work then.
Could you place the link below the the input field as in the interface modal ("Suggest new" link)?

Maybe we can add a popover-modal that explains the used techniques (read: the different name generator settings). This is just an rough idea though and no requirement imho.